### PR TITLE
port toranj test007-traffic-router-sleepy.py

### PR DESCRIPTION
**Pull request by priyankaCh22**

### CODE REVIEW

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import silk.node.fifteen_four_dev_board as ffdb
+import silk.tests.testcase as testcase
+
+hwr.global_instance()
+
+MSG_LENS = [40, 100, 400, 800, 1000]
+
+
+class TestTrafficRouterSleepyEndDevice(testcase.TestCase):
+    """Test traffic between a router and an sleepy-end-device in a two node network.
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: a sleepy-end-device

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        addresses = [
+            (self.router, self.sleepy_end_device, AddressType.LLA, AddressType.LLA),
+            (self.router, self.sleepy_end_device, AddressType.LLA, AddressType.MLA),
+            (self.router, self.sleepy_end_device, AddressType.MLA, AddressType.MLA),
+            (self.sleepy_end_device, self.router, AddressType.LLA, AddressType.LLA),
+            (self.sleepy_end_device, self.router, AddressType.LLA, AddressType.MLA),
+            (self.sleepy_end_device, self.router, AddressType.MLA, AddressType.MLA)
+        ]
+
+        for poll_interval in [10, 100, 300]:
```
Comment (qilinkang) [inline silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: put poll internal list as a constant variable same as MSG_LENS ?

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            self.sleepy_end_device.setprop(wpan.WPAN_POLL_INTERVAL, str(poll_interval))
+            interval = self.sleepy_end_device.getprop(wpan.WPAN_POLL_INTERVAL)
+            self.assertEqual(int(interval), poll_interval)
+
+            for i, (src, dst, src_type, dst_type) in enumerate(addresses):
+                port = random.randint(10000 + i * 100, 10099 + i * 100)
+                timeout = 5
+                delay = 1
```
Comment (tzm41) [inline silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: Do these two variables need to be redefined for each loop cycle? Potentially could be moved outside of the loop.

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py (hunk 1 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: Ack

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py (hunk 2 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: Ack

File: silk/tests/openthread/ot_test_traffic_router_sleepy_device.py (hunk 3 above)
Comment (priyankaCh22) [reply silk/tests/openthread/ot_test_traffic_router_sleepy_device.py]: Ack

**Review by tzm41 — APPROVED** [review]
LGTM

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**